portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,21 +74,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = userService.getCurrentUser().getEmail();
     String comment = getParameter(request, ENTITY_COMMENT_HEADER, /* DefaultValue= */ "");
     long timestamp = System.currentTimeMillis();
+    Optional<Double> sentiment = getSentiment(comment);
```

Comment:
name it sentimentOptional. The Same for storeComments function

Reply:
Done